runtests.py

```diff
@@ -17,3 +18,4 @@
     else:
         args.insert(2, 'openwisp2')
     execute_from_command_line(args)
+    sys.exit(pytest.main(['openwisp_notifications/tests/test_websockets.py']))
```

Comment:
@atb00ker look at this :point_up: 
Should be useful for: https://github.com/openwisp/openwisp-controller/issues/223